src/xrGame/ui/UIMapWnd.h

```diff
@@ -156,10 +156,10 @@ class CUIMapWnd : public CUIWindow, public CUIWndCallback
     };
     void AddMapToRender(CUICustomMap*);
     void RemoveMapToRender(CUICustomMap*);
-    CUIGlobalMap* GlobalMap() { return m_GlobalMap; };
-    const GameMaps& GameMaps() { return m_GameMaps; };
+    CUIGlobalMap* GlobalMap() { return m_GlobalMap; }
+    const GameMaps& getGameMaps() { return m_GameMaps; }
```

Comment:
The same question as to GetLocations

Reply:
Same as GetLocations

Reply:
But that is camelCase

Reply:
PascalCase should be used for publics and globals.